More docs (#146)
siemens/fluffi: getting_started.md

```
sudo ./buildAll.sh
```

> If you have already set up the [FUN](#setting-up-the-fun) AND your compile machine is part of the FUN, you can directly upload your binaries to the FLUFFI ftp server (ftp.fluffi) by calling

Reviewer: I would suggest lower case "and" here and small changes to fit in with the prose style of the rest of the getting started guide
```suggestion
If you have already set up the [FUN](#setting-up-the-fun) and it is reachable from the machine you use to compile, you can directly upload your binaries to the FLUFFI ftp server (ftp.fluffi) by calling
```

buildAll.bat
```

> If you have already set up the [FUN](#setting-up-the-fun) AND your compile machine is part of the FUN, you can directly upload your binaries to the FLUFFI ftp server (ftp.fluffi) by calling

Reviewer: see above

```
buildAll.bat -DEPLOY_TO_FTP TRUE
```

> Please note the FLUFFI binaries MUST have been uploaded to the ftp server at some time, in order for the deployment to work later.

Reviewer: ```suggestion
Please note that the FLUFFI binaries must be uploaded to the ftp server in order for the automatic deployment from the web interface to work.
```
